### ISSUE TITLE
"Operation now in progress" on second mount

### ISSUE THREAD

**Issue by odedia** [Azure/kubernetes-volume-drivers#26]
**What happened**:
Upon a second pod being applied with an SMB mount, I got an error in the log:

```bash
Warning  FailedMount  5s    kubelet, vm-660ec054-707c-4836-4ffb-17b28fc5375f  MountVolume.SetUp failed for volume "test2" : mount command failed, status: Failure, reason: Failed to mount device at /var/lib/kubelet/pods/abf3aef0-7643-11e9-b928-42010a000c32/volumes/microsoft.com~smb/test2, user:odedia, //35.246.33.106/test2, error log:mount: mount //35.246.39.1/test2 on /var/lib/kubelet/pods/abf3aef0-7643-11e9-b928-42010a000c32/volumes/microsoft.com~smb/test2 failed: Operation now in progress
```

**What you expected to happen**:
Second mount should succeed like the first mount

**How to reproduce it**:
Run a pod with direct volume mount, then run a second pod with a different mount. The second pod will fail.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1.12
- OS (e.g. from /etc/os-release): 16.04.5 LTS (Xenial Xerus)
- Kernel (e.g. `uname -a`): Linux 6755f269-791d-49a0-8a47-8a3c3ed287f0 4.15.0-43-generic #46~16.04.1-Ubuntu SMP Fri Dec 7 13:31:08 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
- Install tools:
- Others:

**Comment by andyzhangx** [comment]
could you provide the detailed config with cred removed?
second pod with a different mount but with same smb target on the same node?

**Comment by odedia** [comment]
I just figured it out - it appears that I didn't setup the share itself properly. I tried connecting to it from another machine and it failed again. Once I setup the share with read permission for Everyone it did work from another desktop and from Kubernetes, not sure why it required it but regardless this is not related to the flexvolume itself.
For reference, the issue was a second application entirely pointing to a second smb share.

Thanks for the assistance.
